apps/amiapp_flutter/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -7,11 +7,11 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		0888D1012BEC428D66F8D1ED /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 6615E32455EADFC1FBDD78CC /* GoogleService-Info.plist */; };
```

Comment:
This was updated because....
 https://github.com/customerio/customerio-flutter/pull/105/files#r1427233001